api/src/test/java/org/openmrs/module/metadatadeploy/handler/impl/ConceptDeployHandlerTest.java

```diff
@@ -56,223 +56,192 @@
 	@Autowired
 	private MetadataDeployService deployService;
 
-    @Autowired
-    private ConceptService conceptService;
-
-    private static final String UUID = "f35c9fde-cdc3-11e4-9dcf-b36e1005e77b";
-
-    @Test
-    public void testCreate() {
-        Concept initial = new ConceptBuilder(UUID)
-                .datatype(existing(ConceptDatatype.class, "8d4a4ab4-c2cc-11de-8d13-0010c6dffd0f"))
-                .conceptClass(existing(ConceptClass.class, "a82ef63c-e4e4-48d6-988a-fdd74d7541a7"))
-                .name(new ConceptName("History of present illness", Locale.ENGLISH))
-                .description(new ConceptDescription("How the patient describes it", Locale.ENGLISH))
-                .build();
-        initial.getName().addTag(new ConceptNameTag("refapp-preferred", "preferred in refapp"));
-
-        // Check installing new
-        deployService.installObject(initial);
-
-        Concept created = conceptService.getConceptByUuid(UUID);
-        assertThat(created.getNames().size(), is(1));
-        assertThat(created.getName(), allOf(
-                hasProperty("name", is("History of present illness")),
-                hasProperty("locale", is(Locale.ENGLISH)),
-                hasProperty("localePreferred", is(true)),
-                hasProperty("conceptNameType", is(ConceptNameType.FULLY_SPECIFIED)),
-                hasProperty("tags", containsInAnyOrder(Arrays.asList(
-                                hasProperty("tag", is("refapp-preferred"))
-                        )
-                ))));
-        assertThat(created.getDescriptions().size(), is(1));
-        assertThat(created.getDescription(), allOf(
-                hasProperty("description", is("How the patient describes it")),
-                hasProperty("locale", is(Locale.ENGLISH))));
```

Comment:
Did you make any changes in this method? If so, did you rewrite the whole method?

Reply:
i did not do any changes here and have removed the file that i had pushed